master/internal/experiment/bulk_action.go

```diff
@@ -173,7 +179,7 @@ func editableExperimentIds(ctx context.Context, inputExpIDs []int32,
 	if filters == nil {
 		experimentIDList = inputExpIDs
 	} else {
-		experimentIDList, err = FilterToExperimentIds(ctx, filters)
+		experimentIDList, err = FilterToExperimentIds(ctx, filters, excludedExperimentIds)
```

Comment:
Since `experimentIds` only takes effect when `filters == nil`, it would be nice to have an input validation for related calls, to specifically make sure that when `experimentIds` exists then `filters` should not exists and vice versa.